apiclient/types/model.go

```diff
@@ -12,7 +12,7 @@ type ModelManifest struct {
 	ModelProvider string     `json:"modelProvider,omitempty"`
 	Alias         string     `json:"alias,omitempty"`
 	Active        bool       `json:"active"`
-	Usage         ModelUsage `json:"usage,omitempty"`
+	Usage         ModelUsage `json:"usage"`
```

Comment:
Made this change so the frontend can map to the backend response via enum

Reply:
im explicitly  OKing this. We'll see if a real dev yells at me later.